boards/nxp/frdm_imx93/frdm_imx93_mimx9352_a55.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "frdm_imx93-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "NXP i.MX93 A55";
```

Comment:
how about  `model = "NXP i.MX93 11X11 FRDM board A55 core"`? basically, the model name used in Linux + the differentiation between A55 and the M core

Reply:
rename to "NXP FRDM-IMX93 A55".